api/Dockerfile.skaffold

```diff
@@ -38,7 +38,7 @@ ADD ./src /src/src
 
 RUN make build
 
-# COPY ./kots.so /lib/kots.so
+COPY ./kots.so /lib/kots.so
```

Comment:
no!

Reply:
I'm excited. for the end state of this when these lines no longer exist